src/polyswarm_api/__init__.py

```diff
@@ -8,16 +8,29 @@
 import aiofiles
 import json
 import urllib
+import itertools
 from urllib import parse
 
 from polyswarmartifact import ArtifactType
 
+from . import exceptions
 from .engine_resolver import EngineResolver
 from ._version import __version__, __release_url__
 
 logger = logging.getLogger(__name__)
 
 MAX_HUNT_RESULTS = 20000
+MAX_ARTIFACT_BATCH_SIZE = 256
```

Comment:
What's the max POST size that AI will accept right now? With 256 files at max filesize (32MB) this is going to be sizable. It might be best to also check total data size.

Reply:
I don't think we have a max `POST` size. We do check each individual artifact size for this size
```python
AI_MAX_ARTIFACT_SIZE = int(os.getenv("AI_MAX_ARTIFACT_SIZE", 100 * 1024 * 1024))
```
meaning 100Mb at the moment.

Reply:
Ah, missed responding to this. Yes, let's break that into total post size and max artifact size in AI. If we let a user submit a 100MB file into AI right now, it will accept it but fail to submit the file to `polyswarmd`.

I think we should eventually check and properly chunk submissions here as well, but this is fine for now.